src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not an SCSS file. Please ensure that the stylesheet is compiled to CSS before linking it in the HTML.